[2023-07-11 20:12:19,878] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,656] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,696] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,701] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,704] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,923] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:13,926] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:13] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:18,944] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:18] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:18,972] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:18] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:18,979] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:18] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:18,981] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:18] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:19,103] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:19] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:17:19,106] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:17:19] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:20:42,408] WARNING add_aplicacao() L70   Erro ao adicionar a aplica√ß√£o 'SAP Success Factor', N√£o foi poss√≠vel salva novo item
(OperationalError('(sqlite3.OperationalError) no such table: aplicacao')) - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py L70  
[2023-07-11 20:20:42,409] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:20:42] "[31m[1mPOST /aplicacao HTTP/1.1[0m" 400 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:20:47,561] WARNING add_aplicacao() L70   Erro ao adicionar a aplica√ß√£o 'SAP Success Factor', N√£o foi poss√≠vel salva novo item
(OperationalError('(sqlite3.OperationalError) no such table: aplicacao')) - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py L70  
[2023-07-11 20:20:47,562] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:20:47] "[31m[1mPOST /aplicacao HTTP/1.1[0m" 400 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:21:01,831] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.96:5000 (Press CTRL+C to quit) - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:21:01,833] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:21:38,272] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:21:38] "POST /aplicacao HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:21:55,467] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:21:55] "GET /aplicacao?id=1 HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:22:02,714] ERROR log_exception() L1449 Exception on /aplicacao [GET] - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py", line 221, in get_aplicacao
    if not aplicacao:
           ^^^^^^^^^
UnboundLocalError: cannot access local variable 'aplicacao' where it is not associated with a value
[2023-07-11 20:22:02,717] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:22:02] "[35m[1mGET /aplicacao HTTP/1.1[0m" 500 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:22:09,433] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:22:09] "GET /aplicacao?sigla=SSF HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:26:37,096] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:26:37] "POST /aplicacao HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:27:03,644] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:27:03] "GET /aplicacao?id=1 HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
